src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -238,6 +259,16 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
     }
   }
 
+  private validatePhoneNumber(phoneNumber: string) {
+    const error = validatePresence(phoneNumber)
+    const hasError = error != null
```

Comment:
```suggestion
    const hasError = error !== null
```

Reply:
Maybe a typo but good to know that `!= null` is a weird JS shortcut that means: not equal `null` _and_ not equal `undefined`.